content/en/docs/concepts/storage/volumes.md

```diff
@@ -462,92 +462,155 @@ and then removed entirely in the v1.26 release.
 
 ### hostPath {#hostpath}
 
-{{< warning >}}
-HostPath volumes present many security risks, and it is a best practice to avoid the use of
-HostPaths when possible. When a HostPath volume must be used, it should be scoped to only the
-required file or directory, and mounted as ReadOnly.
-
-If restricting HostPath access to specific directories through AdmissionPolicy, `volumeMounts` MUST
-be required to use `readOnly` mounts for the policy to be effective.
-{{< /warning >}}
-
 A `hostPath` volume mounts a file or directory from the host node's filesystem
 into your Pod. This is not something that most Pods will need, but it offers a
 powerful escape hatch for some applications.
 
+{{< warning >}}
+Using the `hostPath` volume type presents many security risks.
+If you can avoid using a `hostPath` volume, you should. For example,
+define a [`local` PersistentVolume](#local), and use that instead.
+
+If you are restricting access to specific directories on the node using
+admission-time validation, that restriction is only effective when you
+additionally require that any mounts of that `hostPath` volume are
+**read only**. If you allow a read-write mount of any host path by an
+untrusted Pod, the containers in that Pod may be able to subvert the
+read-write host mount.
+
+---
+
+Take care when using `hostPath` volumes, whether these are mounted as read-only
+or as read-write, because:
+
+* Access to the host filesystem can expose privileged system credentials (such as for the Kubelet) or privileged APIs
+  (such as the container runtime socket), that can be used for container escape or to attack other
+  parts of the cluster.
+* Pods with identical configuration (such as created from a PodTemplate) may
```

Comment:
```suggestion
* Pods with identical configuration such as created from the same PodTemplate may
```

Reply:
The suggestion here doesn't read like idiomatic English. Can you clarify what's not right with what I proposed @tengqm?

Reply:
Answered in another comment. I think we may want to avoid parenthesis when possible. They interrupt the sentences and make them don't flow well. It can be used occasionally for reminding something, but having so many parens in these bullets could be a problem. Never read docs like this before.

Reply:
This is moved text; I've left it as-is but changed where it appears in the page.